gen/nvml/lib.go

```diff
@@ -25,7 +30,64 @@ const (
 	nvmlLibraryLoadFlags = dl.RTLD_LAZY | dl.RTLD_GLOBAL
 )
 
-var nvml *dl.DynamicLibrary
+type nvmlLib struct {
+	mu  sync.Mutex
```

Comment:
Is there a particular reason to include the `mu`here?

For types like these, I typically just embed the mutex type so I can do e.g. `nvmlLib.Lock()` instead of `nvmlLib.mu.Lock()`

Reply:
Using `mu` is pretty common in example code. Maybe that's for demonstration purposes though.

I will update to embed the type then.

Reply:
Should be addressed in the latest.